privacyidea/api/lib/postpolicy.py

```diff
@@ -563,66 +575,72 @@ def get_webui_settings(request, response):
             scope=SCOPE.WEBUI,
             realm=realm,
             client=client,
-            active=True
+            active=True,
+            audit_data=g.audit_object.audit_data
         )
         search_on_enter = policy_object.get_policies(
             action=ACTION.SEARCH_ON_ENTER,
             scope=SCOPE.WEBUI,
             realm=realm,
             client=client,
-            active=True
+            active=True,
+            audit_data=g.audit_object.audit_data
         )
         hide_welcome = policy_object.get_policies(
             action=ACTION.HIDE_WELCOME,
             scope=SCOPE.WEBUI,
             realm=realm,
             client=client,
-            active=True
+            active=True,
+            audit_data=g.audit_object.audit_data
         )
         hide_welcome = bool(hide_welcome)
         default_tokentype_pol = policy_object.get_action_values(
             action=ACTION.DEFAULT_TOKENTYPE,
             scope=SCOPE.WEBUI,
             realm=realm,
             client=client,
-            unique=True
+            unique=True,
+            audit_data=g.audit_object.audit_data
         )
         show_seed = policy_object.get_policies(
             action=ACTION.SHOW_SEED,
             scope=SCOPE.WEBUI,
             realm=realm,
             client=client,
-            active=True
+            active=True,
+            audit_data=g.audit_object.audit_data
         )
         show_seed = bool(show_seed)
 
         token_page_size = DEFAULT_PAGE_SIZE
         user_page_size = DEFAULT_PAGE_SIZE
         default_tokentype = DEFAULT_TOKENTYPE
         if len(token_page_size_pol) == 1:
-            token_page_size = int(token_page_size_pol[0])
+            token_page_size = int(token_page_size_pol.keys()[0])
```

Comment:
Same as above, use `int(list(token_page_size_pol)[0])`

Reply:
done. please check the upcoming commit 39f2f33c4a54e7fc9353453dfddcce6bb1b52d98